Dockerfile

```diff
@@ -22,8 +23,8 @@ ENV XC_NUM_LOGROTATE 10
 ENV XC_FIX_DIR_OWNERS yes
 
 # Create the xrootd user with a fixed GID/UID
-RUN groupadd -o -g 10940 xrootd
-RUN useradd -o -u 10940 -g 10940 -s /bin/sh xrootd
+RUN groupadd -o -g 0 xrootd
+RUN useradd -o -u 10940 -g 0 -s /bin/sh xrootd
```

Comment:
What's this for? Does OKD not like groups?

Reply:
By default, OKD runs containers with an ephemeral UID and GID 0.